content/en/docs/components/training/user-guides/fine-tuning.md

```diff
@@ -87,6 +87,9 @@ TrainingClient().train(
 After you execute `train`, the Training Operator will orchestrate the appropriate PyTorchJob resources
 to fine-tune the LLM.
 
+**NOTE**:
+Platform engineers can customize the storage initializer and trainer images by setting the `STORAGE_INITIALIZER_IMAGE` and `TRAINER_TRANSFORMER_IMAGE` environment variables before executing the `train` command. 
```

Comment:
Thank you for adding this!
Please can you add a new section in this doc, so we can reference users to that link

## Use custom images for the Fine-Tuning API